egs2/librimix/enh_diar1/local/run_adapt.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Where is this file used?

Reply:
This is used for fine-tuning the model on the 2&3-speaker mixture dataset. I will add some instructions in README later.

Reply:
Got it!